server/drivers/hermes-os.test.ts

```
@@ -0,0 +1,462 @@
+}
+
+function drain(instance: Awaited<ReturnType<typeof HermesOsDriver["create"]>>, turn: SendTurnInput): Promise<RuntimeEvent[]> {
+  return new Promise(async (resolve, reject) => {
+    const events: RuntimeEvent[] = [];
+    const unsub = instance.adapter.onEvent((e) => events.push(e));
+    try {
+      await instance.adapter.sendTurn(turn);
+    } catch (e) { reject(e); }
+    unsub();
+    resolve(events);
+  });
```

Biome (2.5.6)

[error] Promise executor functions should not be `async`.

(lint/suspicious/noAsyncPromiseExecutor)